tests/api/test_responses.py

```diff
@@ -156,19 +156,6 @@ def test_redirect_response_endpoint(simple_app):
     assert resp.status_code == 302
 
 
-def test_default_object_body(simple_app):
-    app_client = simple_app.app.test_client()
-    resp = app_client.post('/v1.0/test-default-object-body')
-    assert resp.status_code == 200
-    response = json.loads(resp.data.decode('utf-8', 'replace'))
-    assert response['stack'] == {'image_version': 'default_image'}
-
-    resp = app_client.post('/v1.0/test-default-integer-body')
-    assert resp.status_code == 200
-    response = json.loads(resp.data.decode('utf-8', 'replace'))
-    assert response == 1
-
-
```

Comment:
why is this removed?

Reply:
Here's what @badcure had to say about why he removed it:
https://github.com/zalando/connexion/pull/955#issuecomment-495367520
Removed in https://github.com/zalando/connexion/pull/955/commits/cdcb0fda36d1a8dc0a75e47934a67a0c50f4ad6d (part of original PR #955) or https://github.com/zalando/connexion/pull/1054/commits/098854d9276276c225727885bde64335eecfc2fb (in this PR #1054)